### PR TITLE
Include date in run status tooltips

### PR DESCRIPTION
**Pull request by rileyjbauer**
- Last 5 runs status icons in `ExperimentList` will now show the run's start date.
- The status icon at the top of the `RunDetails` page shows start date.
- The status icons for individual steps in the runtime graph will show start and end date.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/671)

### CODE REVIEW

File: frontend/src/pages/ExperimentList.test.tsx [hunk 1]
```diff
@@ -34,6 +35,9 @@ describe('ExperimentList', () => {
   const historyPushSpy = jest.fn();
   const listExperimentsSpy = jest.spyOn(Apis.experimentServiceApi, 'listExperiment');
   const listRunsSpy = jest.spyOn(Apis.runServiceApi, 'listRuns');
+    // We mock this because it uses toLocaleDateString, which causes mismatches between local and CI
```
Comment (yebrahim) [inline frontend/src/pages/ExperimentList.test.tsx]: nit: indentation

**Comment by yebrahim** [comment]
Does this fix https://github.com/kubeflow/pipelines/issues/61?

**Comment by rileyjbauer** [comment]
> Does this fix #61?

Nice catch. Yes it does, although the tooltips are not fully aligned with the ones Ajay provided in that issue. It is probably fine to close the issue regardless

**Pull request merged**